xbmc/pvr/PVRManager.cpp

```diff
@@ -600,7 +603,6 @@ bool CPVRManager::SetWakeupCommand(void)
 
 bool CPVRManager::StartUpdateThreads(void)
 {
-  StopUpdateThreads();
```

Comment:
please elaborate on this. why does stopping a thread before starting it fix a segfault? a stopped thread is a pre-condition for starting it.